tools/fluent/README.md

```diff
@@ -0,0 +1,11 @@
+# Fluent for localizers
+
+<!-- toc -->
+
+Fluent is a localization system, previously known as L20n, designed at Mozilla to unleash the expressive power of the natural language. The format used to describe translation resources used by Fluent is called `FTL`.
+
+This section of documentation includes a brief introduction to Fluent’s syntax for localizers and covers specific details regarding Mozilla’s implementation (branding rules, Gecko BUILT-INs, etc.). For a detailed overview of Fluent, consult the [official documentation](http://projectfluent.org/fluent/guide/).
+
+* [Introduction to Fluent syntax](basic_syntax.md).
+* [Brand names](brand_names.md).
+* [Gecko BUILT-INs](gecko_builtins.md).
```

Comment:
Can we find a less jargon-y title for this? Just `Functions`? Surely not ALL-CAPS, that just scares people, and rightfully so ;-) .

Reply:
`Gecko built-in functions`?